### PR TITLE
Add alias fallback info to keystone and nova extensions.

### PR DESCRIPTION
**Pull request by zack-shoylev**

### CODE REVIEW

File: apis/openstack-keystone/src/main/java/org/jclouds/openstack/keystone/v2_0/extensions/RoleAdminApi.java [hunk 1]
```diff
@@ -48,7 +48,7 @@
  *
  */
 @Beta
-@Extension(of = ServiceType.IDENTITY, namespace = ExtensionNamespaces.OS_KSADM)
+@Extension(of = ServiceType.IDENTITY, namespace = ExtensionNamespaces.OS_KSADM, name = "OpenStack Keystone Admin", alias = "OS-KSADM")
```
Comment (nacx) [inline apis/openstack-keystone/src/main/java/org/jclouds/openstack/keystone/v2_0/extensions/RoleAdminApi.java]: Out of curiosity. Why are these values not used as a constant, like the rest?

Comment (zack-shoylev) [inline apis/openstack-keystone/src/main/java/org/jclouds/openstack/keystone/v2_0/extensions/RoleAdminApi.java]: Keystone only has two extensions, and I wasn't sure how I wanted these implemented. Considering how nova is, I will probably add another file and refactor these to constants.

**Comment by zack-shoylev** [comment]
merged

**Comment by appsdesh** [comment]
Hello!
Looks like your commit helped me in solving following issue
http://www.mail-archive.com/dev@jclouds.apache.org/msg06427.html

I was wondering if you know what changed in Keystone which broke jclouds 1.8 and needed your patch?

**Pull request closed without merging**